common/decklist.cpp

```diff
@@ -466,66 +466,111 @@ bool DeckList::saveToFile_Native(QIODevice *device)
 bool DeckList::loadFromStream_Plain(QTextStream &in)
```

Comment:
I don't plan on coming back to this personally, it'll be someone else small project